extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -78,5 +78,5 @@ private const val LAYER_TYPE_SYMBOL = "symbol"
 private const val STREET_V7 = "mapbox.mapbox-streets-v7"
 private const val STREET_V8 = "mapbox.mapbox-streets-v8"
 private val SUPPORTED_SOURCES = listOf(STREET_V7, STREET_V8)
-private val EXPRESSION_REGEX = Regex("\\[\"get\",\\s*\"(name|name_.{2,7})\"\\]")
+private val EXPRESSION_REGEX = Regex("\\[\"get\",\\s*\"(name_.{2,7})\"\\]")
```

Comment:
do we have a test for this case?

Reply:
No, we don't have method to verify it in test case now